Formula/cmusfm.rb

```diff
@@ -0,0 +1,45 @@
+class Cmusfm < Formula
+  desc "Last.fm standalone scrobbler for the cmus music player"
+  homepage "https://github.com/Arkq/cmusfm"
+  url "https://github.com/Arkq/cmusfm/archive/v0.3.3.tar.gz"
+  sha256 "9d9fa7df01c3dd7eecd72656e61494acc3b0111c07ddb18be0ad233110833b63"
+
+  option "with-debug", "Enable debugging support"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libnotify" => :optional
```

Comment:
We don't really want optional deps as they cause source builds, either remove it or require it if it is useful.

Reply:
Where can I read about the decision to move formulae away from source builds? I was surprised to see the `head` spec trip up the auditor, given the Cookbook makes no mention of this shift in policy.